### PR TITLE
Create scalb.md

### PR DESCRIPTION
**Pull request by Bobliuuu**
### Description

Closes issue #911 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/scalb/scalb.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/scalb/scalb.md:20]:
```suggestion
Math.scalb(value, scale)
```

File: content/java/concepts/math-methods/terms/scalb/scalb.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/scalb/scalb.md:23]:
```suggestion
The parameter `value` is of type `double` and the parameter `scale` is of type `int`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Bobliuuu I did a first review and left a couple of comments.

**Comment by Bobliuuu** [comment]
Thanks @SSwiniarski!

**Review by SSwiniarski — APPROVED** [review]
@Bobliuuu Looks good for second review.

File: content/java/concepts/math-methods/terms/scalb/scalb.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/scalb/scalb.md:2]:
```suggestion
Title: '.scalb()'
```

File: content/java/concepts/math-methods/terms/scalb/scalb.md
Comment (yangc95) [inline content/java/concepts/math-methods/terms/scalb/scalb.md:28]:
```suggestion
// Main.java
public class Main {
```

**Review by yangc95 — COMMENTED** [review]
Just one more addition, looks good to merge after.

Let us know if you need direct assistance passing the checks! If you'd like to try it out yourself, that's also an option.

**Comment by Bobliuuu** [comment]
Yeah, I'm not exactly sure why the checks are failing. Could you let me know why?

**Comment by yangc95** [comment]
@Bobliuuu Live link: https://www.codecademy.com/resources/docs/java/math-methods/scalb
Screen Shot 2022-11-28 at 10 50 38 AM
You're currently an anonymous contributor, but if you'd like to make yourself visible, make sure to add your GitHub username to your Codecademy Profile and make your account public.

cc: @SSwiniarski

**Comment by lucwhite** [comment]
Hello @Bobliuuu,

I hope this message finds you well! My name is Lucas, and I work on the Curriculum team at Codecademy. Our team is considering ways to encourage more individuals to contribute to Codecademy Docs, and improve the contribution experience. We would like to hear from people such as yourself who contribute to Docs and I was wondering if you would be willing to participate in an interview to provide information on your background and experience with Docs contribution? 

The interview will take about an hour and will be conducted over Zoom. Should you agree to participate, we will find a time window using Calendly that works for you within the next three weeks. Let us know if you are interested in participating in an interview or have any questions by emailing me at the address below!

Best, 
Lucas White
lucas@codecademy.com

**Pull request merged**
